app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -14,8 +14,15 @@ def sync_monitor
   end
 
   def sync_from_system
-    ensure_kube_monitors_started
+    # All miq_server instances have to reside on the same Kubernetes cluster, so
+    # we only have to sync the list of pods and deployments once
+    ensure_kube_monitors_started if my_server.is_master?
```

Comment:
Hm so just `is_master?` won't do it because that is only set after initial server startup.  We might have to check if `my_zone` is default (and make sure that server is first)

Reply:
I thinking maybe we encapsulate that `my_zone == "default"` logic into a method